### PR TITLE
Log exception in wsoc 2.1 FAT

### LINKED ISSUE #22404: Handle DeploymentException in Upgrade Test in Wsoc 2.1 FAT
Should we at least log the exception so we know what happened during an exception scenario during testing? I'm fine doing this in a follow-on issue if you open the issue now and w e address it soon.

_Originally posted by @pnicolucci in https://github.com/OpenLiberty/open-liberty/pull/22296#discussion_r964326285_

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #22404

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build

File: dev/io.openliberty.wsoc.2.1.internal_fat/test-applications/basic21.war/src/basic/war/UpgradeServlet.java [hunk 1]
```diff
@@ -12,34 +12,30 @@
...
         ServerContainer container = (ServerContainer) req.getServletContext().getAttribute("jakarta.websocket.server.ServerContainer");
-        try{
-            // echo endpoint is used into another test, so I am reusing it here. 
+        try {
+            // echo endpoint is used into another test, so I am reusing it here.
             container.upgradeHttpToWebSocket(req, resp, ServerEndpointConfig.Builder.create(EchoServerEP.class, "/echo").build(), new HashMap<String, String>());
-        } catch(DeploymentException ex){
-            
+        } catch (DeploymentException ex) {
+            req.getServletContext().log(ex.toString());
```
Comment (volosied) [inline dev/io.openliberty.wsoc.2.1.internal_fat/test-applications/basic21.war/src/basic/war/UpgradeServlet.java:38]: I haven't seen this log call before. That's convenient. I see it just logs here the message here then: https://github.com/OpenLiberty/open-liberty/blob/6a5454d104f304cbae127abce326a8d9d049fbcd/dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebApp.java#L3442 

We don't need any to make any message here right? (or any translations)?

**Review by volosied — APPROVED** [review]

**Review by isaacrivriv — APPROVED** [review]
Looks good, thanks!

File: dev/io.openliberty.wsoc.2.1.internal_fat/test-applications/basic21.war/src/basic/war/UpgradeServlet.java (hunk 1 above)
Comment (pnicolucci) [reply dev/io.openliberty.wsoc.2.1.internal_fat/test-applications/basic21.war/src/basic/war/UpgradeServlet.java:38]: We're just logging a message from a test Servlet so need for any sort of translation. Does that make sense @volosied ?

**Pull request merged**